spec/simplecov_spec.rb

```diff
@@ -270,6 +270,31 @@
       end
     end
 
+    context "with_reports_to_clear_up" do
+      after do
+        clear_mergeable_reports
+      end
+
+      def a_void_method_call_that_wont_warn
+        true
+      end
+
+      it "starts coverage of subprocesses" do
+        SimpleCov.command_name "RSpec"
+        SimpleCov.start
+
+        a_void_method_call_that_wont_warn # this process
+
+        pid = Process.fork do
+          a_void_method_call_that_wont_warn # that process
```

Comment:
It's unclear to me why we need those method calls - to generate data so that we have a result in the end?